workflow-examples/src/main/java/com/redhat/parodos/examples/master/MasterWorkFlowConfiguration.java

```diff
@@ -24,6 +28,35 @@
 @Configuration
 public class MasterWorkFlowConfiguration {
 
+	// Assessment workflow
+	@Bean
+	WorkFlowOption onboardingOption() {
+		return new WorkFlowOption.Builder("onboardingOption", "masterWorkFlow")
+				.addToDetails("An example of a complex WorkFlow with Status checks").displayName("Onboarding")
```

Comment:
nit: I think that `displayName` should be in a new line

Reply:
the spring formatter forced to keep it in the that line